tfx/components/trainer/component.py

```diff
@@ -226,7 +231,8 @@ def trainer_fn(trainer.executor.TrainerFnArgs,
         run_fn=run_fn,
         trainer_fn=trainer_fn,
         custom_config=json_utils.dumps(custom_config),
-        model=output)
+        model=output,
+        model_run=model_run)
```

Comment:
add a todo here 

`# TODO(b/158106209): change the model_run as optional output artifact`